lib/rex/post/meterpreter/packet_dispatcher.rb

```diff
@@ -96,7 +96,7 @@ def on_passive_request(cli, req)
       cli.send_response(resp)
     rescue => e
       send_queue.unshift(resp) if resp
-      elog("Exception sending a reply to the reader request: #{cli.inspect} #{e.class} #{e} #{e.backtrace}")
+      elog("Exception sending a reply to the reader request", error:e)
```

Comment:
This drops the `cli.inspect` details, we'll probably want to keep that

```suggestion
      elog("Exception sending a reply to the reader request: #{cli.inspect}", error:e)
```